src/resources/lib/modeling.py

```diff
@@ -269,7 +269,7 @@ def alphanum_key(key: str) -> List[Union[str, int]]:
         flatten_urls_dict = {
             f"{quality}@{stream}": url
             for quality, urls in files.items()
-            for stream, url in urls.items()
+            for stream, url in list(filter(lambda x: (x[0] != "http"), urls.items()))
```

Comment:
This dictionary comprehension gets too complicated. Let's split it into multiple loops for better readability.